bpf/common/common.h

```diff
@@ -39,13 +39,7 @@
 #define HTTP_CONTENT_TYPE_MAX_LEN 16
 
 volatile const u32 mysql_buffer_size = 0;
-
-enum {
-    k_mysql_query_max = 8192,
-    k_mysql_query_max_mask = k_mysql_query_max - 1,
-    k_mysql_error_message_max = 512,
-    k_mysql_error_message_max_mask = k_mysql_error_message_max - 1
-};
+volatile const u32 postgres_buffer_size = 0;
```

Comment:
I think we likely need one buffer size option for all protocols. What do you think? I think it might be an overkill to ask folks to configure each protocol?

Reply:
The rationale for having it configurable per protocol was that if the workload has heavy traffic across multiple protocols (http, mysql, postgres, etc.), increasing the buffer for all of them could generate significant load, while enabling it for a specific protocol is a safer choice. Also, since the default is 0, it effectively acts as a toggle to enable or disable the feature based on the protocol.

Reply:
ok, cool. I can see this being important, it wasn't obvious since both are SQL, but I can imagine you want less bytes for specific protocol.